pkg/graph/schema.graphql

```diff
@@ -1185,6 +1185,8 @@ Represents a contact associated with a system intake
 type SystemIntakeContact {
   euaUserId: String!
   systemIntakeId: UUID!
+  commonName: String
+  email: String
 }
```

Comment:
Since we have `euaUserId`, `commonName`, and `email`, I wonder if it'd make sense to just have the response contain a `systemIntakeId` and a [UserInfo object](https://github.com/CMSgov/easi-app/blob/bf4f410a5b9aaee77c0e2b0b411b257100d69340/pkg/graph/schema.graphql#L74-L81) (which we use elsewhere).

I could go either way, as this is also sufficiently succinct and readable 🤷